packages/@cdktf/cli-core/src/lib/models/terraform-cli.ts

```diff
@@ -460,3 +480,13 @@ export class TerraformCli implements Terraform {
     return;
   }
 }
+
+export async function findGeneratedConfigurationFile(
```

Comment:
nit: it's not returning the file (as in file path) but rather it's contents
```suggestion
export async function tryReadGeneratedConfigurationFile(
```
But, as Daniel recently said: https://twitter.com/jeffboek/status/1701645550683316504 😂 (so feel free to ignore, if you don't care 😄)

Reply:
👍